core/Inst.hpp

```diff
@@ -32,6 +32,32 @@ namespace olympia
     class Inst {
     public:
 
+        class RenameData{
+            public:
+            void setDestination(unsigned int destination){
+                dest = destination;
+            }
+            void setOriginalDestination(unsigned int destination){
+                original_dest = destination;
```

Comment:
Pls use `uint32_t`

Reply:
Addressed in [163a70b](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/163a70b5862658deca0df16b035bbe59dc841c92)